pkg/operator/vspherecontroller/vspherecontroller.go

```diff
@@ -449,7 +450,26 @@ func (c *VSphereController) updateConditions(
 	}
 
 	updateFuncs := []v1helpers.UpdateStatusFunc{}
+
+	degradeCond := operatorapi.OperatorCondition{
+		Type:   "VMwareVSphereOperatorController" + operatorapi.OperatorStatusTypeDegraded,
```

Comment:
Maybe `VMwareVSphere(Operator?)CheckDegraded` would be better, to clearly distinguish it from `VMwareVSphereControllerDegraded`

Reply:
ack

Reply:
fixed.